hs_core/hydroshare/users.py

```diff
@@ -376,7 +376,8 @@ def get_resource_list(creator=None, group=None, user=None, owner=None, from_date
     if edit_permission:
         if group:
             group = group_from_id(group)
-            q.append(Q(gaccess__resource__in=group.gaccess.edit_resources))
+            q.append(Q(short_id__in=group.gaccess.edit_resources.values_list('short_id',
```

Comment:
Curious to know the reason for this change. Was the existing filter not working or the new one is more efficient?